src/App.tsx

```diff
@@ -15,48 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_ALPHBT = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
+
+function getVisibleGoods(sortType: string, isReversed: boolean): string[] {
```

Comment:
```suggestion
function getVisibleGoods(sortType: string, isReversed: boolean) {
```
TS is smart enough to infer returned types

Reply:
> TS is smart enough to infer returned types

Fixed